etc/tempesta_fw.conf

```diff
@@ -984,6 +984,7 @@
 #       http_host_required true|false;
 #       http_methods [METHOD]...;
 #       http_ct_required true|false;
+#       http_method_override_allowed true|false;
```

Comment:
Please describe the option in the Wiki, preferably with the reference to https://cpdos.org/